lib/Agrammon/OutputFormatter/JSON.pm6

```diff
@@ -56,22 +64,37 @@ sub get-data($model, $outputs, $include-filters) {
     return @records;
 }
 
-sub make-record($fq-name, $output, $model, $raw-value, $var, $filters?) {
+sub make-record($fq-name, $output, $model, $raw-value, $var, $filters?, $language?, $prints?) {
     my $format = $model.output-format($fq-name, $output);
     my $full-value = flat-value($raw-value);
     my $value = ($format  && $full-value.defined) ?? sprintf($format, $full-value)
                                                   !! $full-value;
-    return %(
+
+    # skip if prints filter is set and doesn't match this record
+    my $print = $model.output-print($fq-name, $output);
+    next if not $prints ~~ 'All' and not $print ~~ $prints;
+
+    my %record = %(
         :$format,
-        :print($model.output-print($fq-name, $output)),
+        :$print,
         :order($model.output-order($fq-name, $output)),
-        :labels($model.output-labels($fq-name, $output)),
-        :units($model.output-units($fq-name, $output)),
         :fullValue($full-value),
         :$value,
         :$var,
         :$filters,
     );
+
+    # add language specific strings if language specified
+    if $language and $language ne 'All' {
```

Comment:
Hm, do we really need to support this "All" option? Could we not just say "don't pass it if you want all"?

Reply:
Gone. Removed `All` also from `--prints` above.